packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.stories.tsx

```diff
@@ -35,13 +48,99 @@ Base.args = {
   countryCode: {
     label: 'Country code',
     defaultValue: '+1',
-    options: [
-      { country: 'US', code: '+1' },
-      { country: 'DE', code: '+49' },
-    ],
+    options: Object.keys(countryCodeMap).map((key) => ({
+      country: countryCodeMap[key],
+      code: key,
+    })),
   },
   subscriberNumber: {
     label: 'Subscriber number',
+    placeholder: '202 555 0132',
   },
   validationHint: 'Maximum 15 digits',
 };
+
+export const Invalid = (args: PhoneNumberInputProps) => (
+  <PhoneNumberInput {...args} />
+);
+
+Invalid.args = {
+  ...Base.args,
+  validationHint: 'This number is not valid',
+  subscriberNumber: {
+    ...Base.args.subscriberNumber,
+    invalid: true,
+  },
+};
+
+export const Warning = (args: PhoneNumberInputProps) => (
+  <PhoneNumberInput {...args} />
+);
+
+Warning.args = {
+  ...Base.args,
+  hasWarning: true,
+  validationHint: 'This phone number may incur additional charges',
+};
+
+export const Valid = (args: PhoneNumberInputProps) => (
+  <PhoneNumberInput {...args} />
+);
+
+Valid.args = {
+  ...Base.args,
+  showValid: true,
+  validationHint: 'This phone number looks good',
+};
```

Comment:
We use chromatic.com to catch visual bugs by taking a screenshot of every story. We're billed per story, so we try to combine related stories into one to save costs. 

Could you please merge these into a single "Validations" story, similar to the Input component?

Reply:
Code updated